buildpack.md

```diff
@@ -147,8 +159,9 @@ Executable: `/bin/build <layers[EIC]> <platform[AR]> <plan[ER]>`, Working Dir: `
 | Standard error                           | Logs (warnings, errors)
 | `<layers>/launch.toml`                   | App metadata (see [launch.toml](#launchtoml-toml))
 | `<layers>/build.toml`                    | Build metadata (see [build.toml](#buildtoml-toml))
+| `<layers>/stack-layer.toml`              | Stack layer metadata (see [stack-layer.toml](#stack-layer-content-metadata-toml))
```

Comment:
I feel like it is confusing to merge the inputs/outputs from stack buildpacks into this table with the regular buildpack inputs/outputs. Should we have separate `Buildpack Interface` sections for each type of buildpack?

Reply:
Fixed in 3fcab86